kerl

```diff
@@ -894,6 +903,23 @@ endif
 
 rehash
 ACTIVATE_CSH
+    if [ -n "$KERL_INSTALL_DOCSH" ]; then
```

Comment:
You're right, that's going to be easier. Specifically, it will save some headache with the fact that we need Erlang to be installed and active in order to install docsh.